trinity/main.py

```diff
@@ -79,21 +125,26 @@ def main():
         process.terminate()
 
 
-def ropsten_light_node_sync(log_queue, db_path):
-    """
-    Runs an LES node against the Ropsten network.
-    """
-    setup_queue_logging(log_queue)
+@with_queued_logging
+def run_chain(chain_config, sync_mode):
+    if not is_chain_initialized(chain_config):
+        # TODO: this will only work as is for chains with known genesis
+        # parameters.  Need to flesh out how genesis parameters for custom
+        # chains are defined and passed around.
```

Comment:
Can it just be the json-decoded dict of the standard genesis.json format?

Reply:
Yes, this comment is more to indicate that we need to support a `--genesis` argument for specifying the genesis data.